# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../numpy/lapack-config-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./setup.py.orig	2005-01-06 19:16:13.000000000 -0300
+++ ./setup.py	2005-02-09 11:41:46.851285024 -0300
@@ -33,19 +33,19 @@
 
 # delete all but the first one in this list if using your own LAPACK/BLAS
 sourcelist = [os.path.join('Src', 'lapack_litemodule.c'),
-              #os.path.join('Src', 'blas_lite.c'), 
-              #os.path.join('Src', 'f2c_lite.c'), 
-              #os.path.join('Src', 'zlapack_lite.c'),
-              #os.path.join('Src', 'dlapack_lite.c')
+              os.path.join('Src', 'blas_lite.c'), 
+              os.path.join('Src', 'f2c_lite.c'), 
+              os.path.join('Src', 'zlapack_lite.c'),
+              os.path.join('Src', 'dlapack_lite.c')
              ]
 # set these to use your own BLAS;
 
-library_dirs_list = ['/usr/lib/atlas']
-libraries_list = ['lapack', 'cblas', 'f77blas', 'atlas', 'g2c'] 
+library_dirs_list = [] #['/usr/lib/atlas']
+libraries_list = []    #['lapack', 'cblas', 'f77blas', 'atlas', 'g2c'] 
 
 # set to true (1), if you also want BLAS optimized matrixmultiply/dot/innerproduct
-use_dotblas = 1 
-include_dirs = ['/usr/include/atlas']  
+use_dotblas = 0    # 1 
+include_dirs = []  # ['/usr/include/atlas']  
                    # You may need to set this to find cblas.h
                    #  e.g. on UNIX using ATLAS this should be ['/usr/include/atlas']
 extra_link_args = []
